.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Contribute to Starlight",
+  "build": {
+    "dockerfile": "Dockerfile"
+  },
+  "postCreateCommand": "pnpm install",
+  "waitFor": "postCreateCommand",
+  "customizations": {
+    "codespaces": {
+      "openFiles": ["README.md", "CONTRIBUTING.md"]
```

Comment:
I think it may be helpful to just show the contributor guide here. There’s not much of use in the README:

```suggestion
      "openFiles": ["CONTRIBUTING.md"]
```

Reply:
Sounds good. It also looks a bit weird because of the symlink. Shows diff line marking even though it works and saves modifications to the correct file.

![image](https://github.com/withastro/starlight/assets/3436158/2a5d6030-e297-4429-97a7-1a710c46c460)

Reply:
Awesome. Thanks for the tip!